canvas_sdk/utils/helpers.py

```diff
@@ -0,0 +1,2 @@
+def alphanum_only(string: str) -> str:
+    return "".join([char for char in string if char.isalnum()])
```

Comment:
Why was this needed? Wouldn't this prevent matching LOINC codes which tend to (always?) have a hypen?

Reply:
This was done because ICD-10 codes are stored in the database as alphanumeric (without the period). Good call on the LOINC codes, it looks like those are stored in a different table and do include the hyphen. I can change this to only strip out periods.

Reply:
Oh, your valuesets include periods in the ICD-10 codes. That's a departure from any that we've used so far. I'll ask around for any background on including/excluding them, but for continuity and simplicity let's remove them from the valuesets and not do any transformations on the lookups.

Reply:
I copied them directly from the workflow kit. I'll take a look at home-app to see if those include periods.

Reply:
Ah, weird that there's a departure for 2022. I wonder if it was intentional.

Reply:
I stripped the periods from the ICD10 codes in the ValueSet classes.